autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectCard/ExpandedCourseCard/SectionSelect/InstructionalMethodIcon/InstructionalMethodIcon.tsx

```diff
@@ -11,35 +11,42 @@ interface InstructionalMethodIconProps {
 }
 
 /* eslint-disable react/jsx-key */
-const instructionalMethodIcons = new Map<InstructionalMethod, JSX.Element>([
-  [InstructionalMethod.F2F, <People color="action" fontSize="small" />],
-  [InstructionalMethod.INTERNSHIP, <Work color="action" fontSize="small" />],
-  [InstructionalMethod.NONTRADITIONAL, <Search color="action" fontSize="small" />],
-  [InstructionalMethod.WEB_BASED, <Language color="action" fontSize="small" />],
-  [InstructionalMethod.STUDY_ABROAD, <Flight color="action" fontSize="small" />],
-  [InstructionalMethod.NONE, null],
-  [InstructionalMethod.REMOTE, <Laptop color="action" fontSize="small" />],
-  [InstructionalMethod.F2F_REMOTE_OPTION, (
+const instructionalMethodIcons: Record<InstructionalMethod, JSX.Element> = {
```

Comment:
fyi before making a comment about this: using typescript's Record means that every value must be defined vs. a map where properties are optional. Most places we're using maps we should be using Records instead to prevent dumb mistakes where we forget to set a value